### ISSUE TITLE
"New value type (null) is not in annotated types" despite null coalesce operator.

### ISSUE THREAD

**Issue by donquixote** [kalessil/phpinspectionsea#705]
```php
function foo(string $message) {
  $message = null ?? '';
}
```
gives me "Callable parameter type usage violates definition" - "New value type (null) is not in annotated types".

Interestingly, `$message = null ?: '';` does not create this inspection warning.

**Comment by kalessil** [comment]
That's a known bug: https://youtrack.jetbrains.com/issue/WI-37013 (please vote if you registered there).

**Comment by donquixote** [comment]
ok, thanks

On 18 December 2017 at 21:21, Vladimir Reznichenko <notifications@github.com

**Comment by kalessil** [comment]
No problems =) Hopefully, it'll be fixed soon - get into out bug-tracker pretty often.
